flashinfer/gemm/gemm_base.py

```
@@ -4118,6 +4110,13 @@ def _cudnn_gemm_fp4(
 
 def _cudnn_gemm_fp4_runner():
     class CudnnFp4GemmRunner(TunableRunner):
+        def get_cache_key_extras(self, inputs: List[torch.Tensor]) -> tuple:
+            # inputs layout: a, b, a_descale, b_descale, alpha, out_dtype,
+            #                out, block_size, use_nvfp4, workspace_buffer
+            # All four values affect which cuDNN graph is built.
+            _, _, _, _, alpha, out_dtype, out, block_size, use_nvfp4, _ = inputs
```

Ruff (0.15.6)

[warning] Unpacked variable `out_dtype` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)